appmesh_models_override/setup.sh

```diff
@@ -4,16 +4,33 @@ set -e
 
 mkdir -p ./vendor/github.com/aws
 
+SDK_MODEL_SOURCE=https://raw.githubusercontent.com/aws/aws-app-mesh-roadmap/master/appmesh-preview/sdk
 SDK_VENDOR_PATH=./vendor/github.com/aws/aws-sdk-go
+API_VERSION=2019-01-25
+API_PATH=$SDK_VENDOR_PATH/models/apis/appmesh/$API_VERSION
+SERVICE_NAME=$([ "$APPMESH_PREVIEW" == "1" ] && echo "appmesh-preview" || echo "appmesh" )
```

Comment:
Does not really impact this CR as this was the case before as well. Will `appmesh` service endpoint support `appmesh-preview` source model at line#7 or we expect some features may not be supported?

Reply:
The App Mesh production endpoint will not support all features in preview, but preview will support all features from App Mesh production. In effect, the preview SDK long-term will be a superset of production.

For features not supported in production, the controller will either error with a 404 (API doesn't exist) or error with a 403 (access denied). We protect new features with feature flags until they're ready to go GA, so it's generally safe to vend the preview models for production use.

Reply:
Perfect, makes sense. Thanks!